### DISCUSSION TITLE
replace duplicated photos

### DISCUSSION THREAD

**Discussion by krzemyczek** [simulot/immich-go#406] [Q&A]
Hi, i have issue with having a lot of photos with wrong date after i uploaded them to immich before finding out this program. so now when i upload the same photos (with correct date i can either not upload duplicate one or upload all of them and finding afterwards which is wrong. Is there way to re upload duplicated photos? Thanks in advance!

**Comment by Chrissica06** [comment]
Did you find any info on this?  I am in the same boat

**Comment by simulot** [comment]
immich-go has evolved a lot since them.
You can share the logs if you have questions
